pkg/server/server.go

```diff
@@ -149,6 +150,10 @@ func (s *Server) ListenAndServe(ctx context.Context, logger log.Logger, port str
 			pprof.Profile(w, r)
 			return
 		}
+		if r.URL.Path == "/debug/pprof/fgprof" {
+    	fgprof.Handler().ServeHTTP(w, r)
+			return
+		}
```

Comment:
This isn't correctly formatted, by the looks of it.
Can you locally run `gofmt -w pkg/server/server.go`, so it'll do that for you. 
Sorry, for not spotting this one earlier.